pilates/activitysim/preprocessor.py

```diff
@@ -55,19 +55,22 @@ def _load_raw_beam_skims(settings):
     return skims
 
 
-def _create_skim_object(data_dir):
+def _create_skim_object(data_dir, overwrite=False):
```

Comment:
You are right about it. I need to add this overwrite in order to incorporate the fresh BEAM skims produced on the BEAM step.

Reply:
If you have any thoughts about the best way to handle intermediate outputs I would love to hear them. 

Previously we were reading/writing from/to cloud storage, and since disk space was therefore never an issue we were archiving all of the intermediate outputs (e.g. skims_2015.csv.gz, skims_2020.csv.gz, skims_2025.csv.gz etc.). It's useful to have access to these files for debugging, validation, or plotting simulation outputs as they evolve throughout the course of a full run (2010 to 2050). But that will quickly eat up disk space now that we're reading/writing locally.

Reply:
We have now the following:
1. The full BEAM skim file that contains all pre-calculated Origin-Destination pairs. This file is pretty big.
2. On each iteration BEAM creates current iteration skims. That also may be a big file but is smaller compared to the full BEAM skims.
3. This current iteration skims are merged to the full BEAM skims. And then this file is used by the ASIM.

All these iteration skim files (along with the other beam output) are stored in the beam output folder. And one can use them for debugging/etc.